tiingo/wsclient.py

```diff
@@ -0,0 +1,93 @@
+import os
+import websocket
+try:
+    import thread
+except ImportError:
+    import _thread as thread
+import time
+import json
+
+GLOB_config=None
+GLOB_on_msg_cb=None
+
+class genericWebsocketClient:
+    '''
+    the methods passed to websocketClient have to be unbounded if we want WebSocketApp to pass everything correctly
+    see websocket-client/#471
+    '''
+    def on_message(ws, message):
+        GLOB_on_msg_cb(message)
+    def on_error(ws, error):
+        print(error)
+    def on_close(ws):
+        pass
+    def on_open(ws):
+        def run(*args):
+            print(GLOB_config)
+            ws.send(json.dumps(GLOB_config))
+        thread.start_new_thread(run, ())
+    def __init__(self,config,on_msg_cb):
+        global GLOB_config
+        global GLOB_on_msg_cb
+        GLOB_config=config
+        GLOB_on_msg_cb=on_msg_cb
+        return
+
+class TiingoWebsocketClient:
+    '''
+    from tiingo import TiingoWebsocketClient
+    def cb_fn(msg):
+        print(msg)
+    subscribe = {
+            'eventName':'subscribe',
+            'authorization':'API_KEY_GOES_HERE',
+            'eventData': {
+                'thresholdLevel':5
+          }
+    }
+    client=TiingoWebsocketClient(subscribe,endpoint="iex",on_msg_cb=cb_fn)
+    while True:pass
+    '''
+
+    def __init__(self,config={},endpoint=None,on_msg_cb=None):
```

Comment:
What do you think about documenting the expected.permitted options for the config dictionary + the expected values for endpoint + the function signature for `on_msg_cb` ? The example code you provided in the docstring is a good start for a documentation page, but we would want to aim for something closer to a descriptive docstring for a class comment. See https://realpython.com/python-pep8/#documentation-strings for more background.